src/open_inwoner/plans/views.py

```diff
@@ -266,12 +266,29 @@ def crumbs(self):
     def get_context_data(self, **kwargs):
         data = super().get_context_data(**kwargs)
 
+        interests = self.request.user.get_interests()
+        plan_templates = PlanTemplate.objects.all()
+
+        # Put the plan templates with interests of the user in front
+        plan_templates = (
+            PlanTemplate.objects.annotate(
+                has_interest=Case(
+                    When(related_categories__name__in=interests, then=Value(1)),
+                    default=Value(0),
+                    output_field=IntegerField(),
+                )
+            )
+            .order_by("-has_interest")
```

Comment:
```suggestion
            .order_by("-has_interest", "id")
```
I would add this to ensure that the sorting is stable, i.e. the relative order of items is preserved. Currently the order of items even without interests is changed by this (at least in the browser, though not in the test for some reason).